source/includes/aws/_internetgateways.md

```diff
@@ -214,4 +214,39 @@ Detach an internet gateway from a VPC in a given [environment](#administration-e
 | Required                                         | &nbsp;                                |
 |--------------------------------------------------|---------------------------------------|
 | `vpcId`<br/>*string*                             | The ID of the VPC.                    |
-| `deleteRoutes`<br/>*boolean*                     | Whether or not to delete the routes attached to the Internet Gateway. Default to true. |
\ No newline at end of file
+| `deleteRoutes`<br/>*boolean*                     | Whether or not to delete the routes attached to the Internet Gateway. Default to true. |
+
+<!-------------------- DELETE AN INTERNET GATEWAY -------------------->
+
+#### Delete an Internet Gateway
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/aws/aws-tesv/internetgateways/igw-00b5e5bb6cce152cb?operation=delete"
```

Comment:
i'm getting an` internetGateway 'igw-0d725a79ba2de538c' has dependencies and cannot be deleted` error in my console when I try this 👀  is that expected under some conditions?

Reply:
Good question, I never encounter that issue before, will need to look into it.

Reply:
Was it in dev? Can you link me the request itself?

Reply:
it was on my local, i'll try finding one in dev

Reply:
here's one on dev that errors out `https://portal.dev.cloudmc.io/api/v1/services/aws-aaaa/cloudmc-aws-dev/internetgateways/igw-0389447eadb99ba3d?operation=delete`

Reply:
I just tried it and it worked on my side damnit.

Reply:
oh yeah I am mistaken, it worked but the task failed!

Reply:
Created a JIRA ticket for it: https://cloudops.atlassian.net/browse/MC-18280

Reply:
Here's a PR: https://github.com/cloudops/cloudmc-aws-plugin/pull/291

Reply:
awesome!! lgtm then :))